src/twisted/internet/defer.py

```diff
@@ -160,6 +171,151 @@ def maybeDeferred(f, *args, **kw):
 
 
 
+def extractCurrentResult(deferred, raises=True):
+    """
+    Extract the current result from a L{Deferred} that has already fired all of
+    its callbacks.
+
+    Take a deferred that has already fired all of its callbacks and extract and
+    return the result of the L{Deferred}. This process of extracting the result
+    will consume the result, error or otherwise. If the L{Deferred} has an
+    error then the process of extracting the result will, by default, re-raise
+    the original exception unless the L{raises} parameter is set to L{False}.
+
+    The primary purpose of this function is to allow code to be written that is
+    internally asynchronous but which can be consumed without an event loop in
+    a synchronous fashion by using L{extractCurrentResult} at the boundaries.
+    This enables libraries to be written that can function both with Twisted
+    and with synchronous code instead of requiring an asynchronous and a
+    synchronous implementation for all functionality.
+
+    @attention: This function CANNOT take a Deferred with pending asynchronous
```

Comment:
Apparently pydoctor doesn't like `@attention`:

```
found unknown field on 'twisted.internet.defer.extractCurrentResult': <Field 'attention' None [u'This function CAN...']>
found unknown field on 'twisted.internet.defer.observeCurrentResult': <Field 'attention' None [u'This function CAN...']>
```

Which is too bad, since epydoc itself seems okay with this, at least according to <http://epydoc.sourceforge.net/fields.html>.

I suppose the thing to do is leave this as-is and file a pydoctor bug to make this work.